### PR TITLE
Add namespace support to USD import file translators.

### PR DESCRIPTION
**Pull request by ppt-adsk**

### CODE REVIEW

File: plugin/adsk/plugin/importTranslator.h [hunk 1]
```diff
@@ -48,6 +48,7 @@ class UsdMayaImportTranslator : public MPxFileTranslator
 
     bool haveReadMethod() const override { return true; }
     bool haveWriteMethod() const override { return false; }
+    bool haveNamespaceSupport() const override { return true; }
```
Comment (ppt-adsk) [inline plugin/adsk/plugin/importTranslator.h:51]: This is the trivial fix.

File: plugin/pxr/maya/lib/usdMaya/importTranslator.h [hunk 2]
```diff
@@ -48,6 +48,7 @@ class UsdMayaImportTranslator : public MPxFileTranslator
 
     bool haveReadMethod() const override { return true; }
     bool haveWriteMethod() const override { return false; }
+    bool haveNamespaceSupport() const override { return true; }
```
Comment (ppt-adsk) [inline plugin/pxr/maya/lib/usdMaya/importTranslator.h:51]: The ADSK import file translator is a copy-paste of the original Pixar import translator, so decided to do the fix there as well, even though no one has complained about it.  I think it is quite arguably a latent bug.  I decided not to refactor the two files and factor out the class to a common one, as the Pixar plugin should become deprecated soon.

File: test/lib/mayaUsd/fileio/testImportWithNamespace.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env mayapy
```
Comment (ppt-adsk) [inline test/lib/mayaUsd/fileio/testImportWithNamespace.py:1]: Test Driven Development: without the fix the test fails, after adding the fix the test passes.

File: test/lib/mayaUsd/fileio/testPrimReader.py [hunk 4]
```diff
@@ -33,24 +33,18 @@ def Read(self, context):
         sphere = UsdGeom.Sphere(usdPrim)
         cmds.polySphere(r=sphere.GetRadiusAttr().Get(), name=usdPrim.GetName())
         MayaNode = context.GetMayaNode(usdPrim.GetPath().GetParentPath(), True)
-        print("primReaderTest.Read Ok")
```
Comment (ppt-adsk) [inline test/lib/mayaUsd/fileio/testPrimReader.py:36]: Unrelated changes, apologies, could not help cleaning this up while looking at files in this directory.

File: test/lib/mayaUsd/fileio/testImportWithNamespace.py [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+from maya import cmds
+from maya import standalone
+
+from pxr import Usd, UsdGeom
+
+import fixturesUtils
+
+import unittest
+
+class testReadWriteUtils(unittest.TestCase):
```
Comment (pierrebai-adsk) [inline test/lib/mayaUsd/fileio/testImportWithNamespace.py:27]: Name of the class seems a bit disconnected with the actual test about namespace.

Comment (ppt-adsk) [reply test/lib/mayaUsd/fileio/testImportWithNamespace.py:27]: Ack!  Sorry about that...  Copy-paste damage.  Will fix once you're done with the review and the build has completed.

**Comment by pierrebai-adsk** [comment]
LGTM except for the test class name, trivial.

**Review by pierrebai-adsk — APPROVED** [review]

**Comment by ppt-adsk** [comment]
@JGamache-autodesk I added you as a reviewer because of the impact on import tests.

File: test/lib/usd/translators/testUsdExportUVSetMappings.py [hunk 6]
```diff
@@ -102,7 +102,7 @@ def testExportUVSetMappings(self):
             mayaMesh = OM.MFnMesh(dagPath.node())
             expected_uvs.append(("pPlane%iShape" % i, mayaMesh.getUVSetNames()))
 
-        expected_sg = set(cmds.ls(type="shadingEngine"))
+        expected_sg = set([sg if sg.startswith('initial') else "Test:"+sg for sg in cmds.ls(type="shadingEngine")])
```
Comment (ppt-adsk) [inline test/lib/usd/translators/testUsdExportUVSetMappings.py:105]: This is getting a little bit out of hand...

**Review by JGamache-autodesk — APPROVED** [review]
Thanks fixing this and for not choosing to remove a few "namespace" parameters from import calls! Looks good to me, even the slightly out of hand part.

**Review by pierrebai-adsk — APPROVED** [review]

**Comment by ppt-adsk** [comment]
All customer comments received so far indicate that no backward compatibility for previous buggy behavior is needed, so we can proceed with merging this change.
